### PR TITLE
Added neighbor info to blocks and phased out CHAIN_COUNT and NETWORK_ID env vars

### PR DESCRIPTION
**Pull request by MRVDH**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207266390671306
  - https://app.asana.com/0/0/1207265113893288

### CODE REVIEW

File: packages/apps/graph/src/devnet/deployment/helper.ts [hunk 1]
```diff
@@ -27,7 +28,7 @@ const getClient = (): IClient => {
   if (!client) {
     client = createClient(
       ({ chainId }) =>
-        `${dotenv.NETWORK_HOST}/chainweb/0.0/${dotenv.NETWORK_ID}/chain/${chainId}/pact`,
+        `${dotenv.NETWORK_HOST}/chainweb/0.0/${networkData.networkId}/chain/${chainId}/pact`,
```
Comment (nil-amrutlal-dept) [inline packages/apps/graph/src/devnet/deployment/helper.ts:31]: We could also probably replace the version `0.0` by `networkData.apiVersion` ?

**Review by nil-amrutlal-dept — COMMENTED** [review]
Marmalade Simulation and Deployment are not working (discussed in person)

**Comment by nil-amrutlal-dept** [comment]
Command `pnpm simulate:marmalade` still does not work on my end. Can you please confirm

**Comment by MRVDH** [comment]
> pnpm simulate:marmalade

Fixed now 👍

File: packages/apps/graph/src/devnet/deployment/marmalade/deploy.ts [hunk 2]
```diff
@@ -36,6 +37,10 @@ export async function deployMarmaladeContracts(
   codeFileDestinationPath: string = marmaladeLocalConfig.codeFilesPath,
   nsDestinationPath: string = marmaladeLocalConfig.namespacePath,
 ): Promise<void> {
+  await initializeNetworkConfig();
+
+  argumentConfig.network = networkData.networkId;
+
```
Comment (nil-amrutlal-dept) [inline packages/apps/graph/src/devnet/deployment/marmalade/deploy.ts:43]: nice

**Review by nil-amrutlal-dept — APPROVED** [review]

**Pull request merged**